code/__DEFINES/tools.dm

```diff
@@ -0,0 +1,4 @@
+#define TOOL_CROWBAR 1
+#define TOOL_SCREWDRIVER 2
+#define TOOL_WIRECUTTER 4
+#define TOOL_WRENCH 8
```

Comment:
No real need to make this a bitflag since the tools conflict with each other. No point in having more than one quality set at a moment.